
Mon, 27 Dec 2010 19:50:32 GMT

COMMAND LINE:
/usr/local/src/pypy-1.4.1-linux64/bin/pypy nbody.pypy-4.pypy 500000

PROGRAM FAILED 


PROGRAM OUTPUT:

Traceback (most recent call last):
  File "app_main.py", line 53, in run_toplevel
  File "nbody.pypy-4.pypy", line 51, in <module>
    PAIRS = list(itertools.combinations(SYSTEM, 2))
AttributeError: 'module' object has no attribute 'combinations'
